### PR TITLE
Stop consuming TriggeredV3 [DO NOT MERGE]

### PR DESCRIPTION
**Pull request by laurence-bird**

### CODE REVIEW

File: src/main/scala/com/ovoenergy/orchestration/kafka/consumers/EventConverter.scala [hunk 1]
```diff
@@ -7,17 +7,6 @@ import com.ovoenergy.comms.templates.util.Hash
 
 object EventConverter {
```
Comment (ZsoltBalvanyos) [inline src/main/scala/com/ovoenergy/orchestration/kafka/consumers/EventConverter.scala]: the conversion of MetadataV2ToV3 can be deleted as well

Comment (laurence-bird) [reply src/main/scala/com/ovoenergy/orchestration/kafka/consumers/EventConverter.scala]: 👍 done!

Comment (laurence-bird) [reply src/main/scala/com/ovoenergy/orchestration/kafka/consumers/EventConverter.scala]: 👍 done

**Comment by ovo-comms-circleci** [comment]
This PR was successfully deployed to production by [CircleCI build 531](https://circleci.com/gh/ovotech/comms-orchestration/531) at 2018-10-01 16:28 BST.

**Pull request merged**